miio/viomivacuum.py

```diff
@@ -174,19 +174,15 @@ class ViomiMopMode(Enum):
 
 class ViomiVacuumStatus:
     def __init__(self, data):
-        # ["run_state","mode","err_state","battary_life","box_type","mop_type","s_time","s_area",
-        # [ 5,          0,     2103,       85,            3,         1,         0,       0,
-        # "suction_grade","water_grade","remember_map","has_map","is_mop","has_newmap"]'
-        # 1,               11,           1,            1,         1,       0          ]
-        self.data = data
+        self._data = data
```

Comment:
Please keep `self.data` naming as this would be otherwise the only status container with a different variable naming.

Reply:
fixed